mbed_targets/__init__.py

```diff
@@ -1,4 +1,24 @@
-"""Exposes the primary interfaces for the library."""
+"""mbed-targets provides an abstraction layer for boards and platforms supported by Mbed OS.
 
+This package is intended for use by developers of Mbed OS.
+
+To fetch data about private targets, an authentication token must be provided in an
+environment variable named `MBED_API_AUTH_TOKEN`. The package also accepts a `.env` file
```

Comment:
I know we tested this and I can't remember what the answer was, which one trumps if both are present?

Reply:
The `.env` file wins as we use `override_system=true` - I'll add clarification on that